skipruntime-ts/core/src/skipruntime_api.ts

```diff
@@ -276,6 +276,17 @@ export type CollectionUpdate<K extends TJSON, V extends TJSON> = {
   isInitial?: boolean;
 };
 
+/**
+ * A `ReactiveResponse` contains metadata which can be used to initiate and manage reactive
+ * connections to/from a service.  It specifies a `collection` and a current `watermark`,
+ * and is sent by Skip services in the `"x-reactive-response"` HTTP header by default,
```

Comment:
Not specific to this PR, but my understanding is that the `X-` prefix on headers has been deprecated for a long time now: https://datatracker.ietf.org/doc/html/rfc6648. And while the headers are case insensitive, I think the most common practice is to capitalize them, such as `X-Reactive-Response`.

Reply:
How do you feel about `Skip-Reactive-Response`?  Not specific to this PR indeed, but as good a time as any to fix it!

Reply:
Yes, we should use the opportunity to find proper names for the headers (they weren't named carefully).
Since it is Skip-specific, I would suggest to name them `Skip-Something`. `Skip-Response-Token`/`Skip-Reactive-Token`/`Skip-Subscription-Token`? The name Skip is a bit confusing here, but that's life.